### ISSUE TITLE
[1.4.6-r2 (38)] Asus Zenfone Go 4.5 / Android 5.1 - Clicking on Back navigates to Home page - From Motifs page click on Back button navigates user to Home page

### ISSUE THREAD

**Issue by hiteshbheda** [Lomotif/lomotif-android-issues#398]
Frequency: Every time

Action:
1. Launch the app
2. Click on "START" button
3. Go to Motifs tab on Add Photos & Videos page
4. Click on Back button
5. Check user is redirected to Albums tab or not

Expected result: User should be redirected to Albums tab, User should not be redirected to Home screen without message

Actual result: User navigates to Home screen

**Comment by xiuie** [comment]
@hiteshbheda May be I don't understand the report, but does the above work like iOS? Tapping on "My Videos" allows you to access albums folder, but hitting back should bring you to Home screen?

**Comment by vysohanych** [comment]
@xiuie I can reproduce it on build 1.4.7 (43) on LG Nexus 5X Android 6.0.1 but I think it's not a bug

**Comment by hiteshbheda** [comment]
@xiuie I can reproduce this issue for build 1.4.7 (43) on Asus Zenfone Go / Android 5.1

**Comment by hiteshbheda** [comment]
@xiuie Yes, actually when user at the motif tab and then click on Back button then he/she should be navigated to Add Photos & Videos screen Album tab, Instead user navigates to Home screen directly. 

And if user navigates to user then one alert Message should be displayed.

**Comment by testershub** [comment]
@xiuie I can reproduce it on build 1.4.7 (43) on Samsung Galaxy S6- Android 6.0.1 but I also think it's not a bug and working as expected.
